src/man/sssd.conf.5.xml

```diff
@@ -733,26 +733,24 @@
                             When SSSD switches to offline mode the amount of
                             time before it tries to go back online will
                             increase based upon the time spent disconnected.
-                            This value is in seconds and calculated by the
-                            following:
-                        </para>
-                        <para>
-                             offline_timeout + random_offset
-                        </para>
-                        <para>
-                            The random offset value is from 0 to 30.
+                            By default SSSD uses incremental behaviour to
+                            calculate delay in between retries.
+                            The longer backend is offline, the bigger delays
+                            in between attempts to checking it status again.
                             After each unsuccessful attempt to go online,
                             the new interval is recalculated by the following:
                         </para>
                         <para>
-                            new_interval = (old_interval * 2) + random_offset
+                             delay = (delay * 2) % offline_timeout_max + (rand() % refresh_max_random_offset)
```

Comment:
Hi,

I would suggest to use the `new_` and `old_` prefixes as in the old version. While the current version is correct from a programming language point of view I think using `new_` and `old_` would be more easy to understand for the common reader.

The first part should be better something like `Minimum(old_delay * 2, offline_timeout_max)`. Using `%` here would mean that we will switch back to a short timeout again after we have reached the maximum.

For the second part I would not use the name of the function we use but something more generic like `random[0..refresh_max_random_offset]`.

I think it would be better to use the `offline_timeout_` prefix for the new option name.

bye,
Sumit

Reply:
Thank you for the suggestion, I have implemented the change you proposed. Please take a look.